test/specs/components/Label/Label-test.tsx

```diff
@@ -7,15 +7,13 @@ import { mountWithProvider } from '../../../utils'
 describe('Label', () => {
   isConformant(Label)
```

Comment:
You should also test `implementsShorthandProp`, see #112.

Reply:
Added implementsShorthandProp test for the image and icon, please check.